### ISSUE TITLE
missing dir@staff in select measures in Altenburg concerto in C major

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/sample-encodings#29]
In the encoding:
https://github.com/music-encoding/sample-encodings/blob/master/MEI_3.0/Music/Complete_examples/Altenburg_concerto_C_major.mei

Staff numbers are missing on dir elements in measures 1, 37, 38, 46, 53, 77, 126, and 128.

For example, measure 53:

```xml


<measure n="53" xml:id="d6409e26747">
   <staff n="1">
      <layer n="1">
         <rest xml:id="d6409e26765" dur="1"/>
      </layer>
   </staff>
   <staff n="2">
      <layer n="1">
         <rest xml:id="d6409e26790" dur="16"/>
         <beam>
            <note xml:id="d6409e26803" pname="g" oct="5" dur="16"/>
            <note xml:id="d6409e26819" pname="e" oct="5" dur="16"/>
            <note xml:id="d6409e26835" pname="c" oct="5" dur="16"/>
            <rest xml:id="d6409e26851" dur="16"/>
            <note xml:id="d6409e26859" pname="g" oct="5" dur="16"/>
            <note xml:id="d6409e26875" pname="f" oct="5" dur="16"/>
            <note xml:id="d6409e26891" pname="d" oct="5" dur="16"/>
         </beam>
      </layer>
   </staff>
   <staff n="3">
      <layer n="1">
         <note xml:id="d6409e26925" pname="c" oct="5" dur="4"/>
         <beam>
            <note xml:id="d6409e26937" pname="d" oct="5" dur="8"/>
            <note xml:id="d6409e26954" pname="g" oct="4" dur="8"/>
         </beam>
      </layer>
   </staff>
   <staff n="4">
      <layer n="1">
         <beam>
            <note xml:id="d6409e26990" pname="c" oct="4" dur="8" dots="1"/>
            <note xml:id="d6409e27005" pname="e" oct="4" dur="16"/>
         </beam>
         <beam>
            <note xml:id="d6409e27019" pname="g" oct="4" dur="8"/>
            <note xml:id="d6409e27033" pname="g" oct="3" dur="8"/>
         </beam>
      </layer>
   </staff>
   <staff n="5">
      <layer n="1">
         <rest xml:id="d6409e27060" dur="1"/>
      </layer>
   </staff>
   <staff n="6">
      <layer n="1">
         <rest xml:id="d6409e27081" dur="1"/>
      </layer>
   </staff>
   <staff n="7">
      <layer n="1">
         <rest xml:id="d6409e27102" dur="1"/>
      </layer>
   </staff>
   <staff n="8">
      <layer n="1">
         <beam>
            <note xml:id="d6409e27132" pname="c" oct="3" dur="8"/>
            <note xml:id="d6409e27146" pname="c" oct="3" dur="16"/>
            <note xml:id="d6409e27162" pname="c" oct="3" dur="16"/>
         </beam>
         <beam>
            <note xml:id="d6409e27178" pname="g" oct="2" dur="8"/>
            <note xml:id="d6409e27192" pname="g" oct="2" dur="16"/>
            <note xml:id="d6409e27208" pname="g" oct="2" dur="16"/>
         </beam>
      </layer>
   </staff>
   <dir tstamp="1" place="above" staff="1">
      <rend fontfam="Times New Roman" fontsize="12" fontweight="bold">Andante</rend>
   </dir>
   <dir tstamp="1" place="below" staff="4">
      <rend fontfam="Times New Roman" fontsize="12" fontweight="bold">Andante</rend>
   </dir>
   <dynam tstamp="1.25" place="below" staff="4">
      <rend>mf</rend>
   </dynam>
   <dynam tstamp="1" place="below" staff="5">
      <rend>mf</rend>
   </dynam>
   <trill tstamp="2" place="above" staff="5" startid="#d6409e26937"/>
   <dynam tstamp="1" place="below" staff="6">
      <rend>mf</rend>
   </dynam>
   <dir tstamp="1" place="above" staff="6">
      <rend fontfam="Times New Roman" fontsize="12" fontweight="bold">Andante</rend>
   </dir>
   <dynam tstamp="1" place="below">
      <rend>p</rend>
   </dynam>
   <dir tstamp="1" place="above">
      <rend fontfam="Times New Roman" fontsize="12" fontweight="bold">Andante</rend>
   </dir>
</measure>
```

The last `dir` elements is missing a `staff` attribute:

```xml
   <dir tstamp="1" place="above">
      <rend fontfam="Times New Roman" fontsize="12" fontweight="bold">Andante</rend>
   </dir>
```

Also the `<dynam>` just above the last dir does is not attached to a `@staff` (or `@startid`):

```xml
   <dynam tstamp="1" place="below">
      <rend>p</rend>
   </dynam>
```

**Comment by pe-ro** [comment]
This example is fubar-ed.  I fixed the issues noted, but there are other problems with movement indications, time signatures, placement of directives, and so on, that my brain is too tired to deal with at the moment.

It seems this is a case of visual, rather than semantic encoding from the outset, which is a royal pain.

**Comment by pe-ro** [comment]
Fixed by 410e6f3
